### PR TITLE
refactor: sdk runtimes

### PR DESCRIPTION
**Pull request by Daanvdplas**
Refactor to use the westend runtimes, for the relay and asset hub (default), for the xcm integration tests. In order to use the paseo runtimes there is still the `paseo` feature.

- First commit adds the westend runtimes. 
- Second commit restructures the crate structure and removes duplicate code for the relay runtimes.

Run paseo tests with `cargo test --no-default-features --features paseo`

Note: left the `polkadot` configuration just in case.

### CODE REVIEW

**Review by al3mart — COMMENTED** [review]
When trying to run the integration tests activating `paseo` feature I get a bunch of compilation errors.

Aside from that everything looks alright!

File: integration-tests/src/lib.rs [hunk 1]
```diff
@@ -1,65 +1,67 @@
...
 			PopNetwork,
 		],
 		bridge = ()
 	},
 }
 
 decl_test_sender_receiver_accounts_parameter_types! {
-	PaseoRelay { sender: ALICE, receiver: BOB },
-	AssetHubPaseoPara { sender: ALICE, receiver: BOB },
+	RelayRelay { sender: ALICE, receiver: BOB },
```
Comment (chungquantin) [inline integration-tests/src/lib.rs:43]: Why not just `Relay` or `PolkadotRelay`?

**Review by chungquantin — COMMENTED** [review]
Westend and mainnet integration tests work for me. Got bunches of compilation issues like what Ale mentioned. Aside from that, no concern with the code changes.

File: integration-tests/src/lib.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (Daanvdplas) [reply integration-tests/src/lib.rs:43]: Yes this is something that can be refactored when we separate the tests in its own folder. I suggest we do this once the integration tests are required for AH as reserve

[the remark this answers is not in the document]
Comment (chungquantin) [reply integration-tests/src/lib.rs:43]: Maybe `UniversalRelay`, like `UniversalLocation` 🙂

**Review by chungquantin — APPROVED** [review]
Paseo test works. Approved.

**Review by al3mart — APPROVED** [review]

**Pull request merged**